content/applications/hr/recruitment/recruitment-flow.rst

```diff
@@ -0,0 +1,189 @@
+================
+Recruitment flow
+================
+
+When a prospective employee applies for a job in Odoo, there is a pre-configured process from the
+:ref:`initial inquiry <recruitment/new>` to :ref:`creating a new employee
+<recruitment/new-employee>` once hired. The following outlines the default recruitment *flow*, or
+sequence of events, for Odoo's *Recruitment* application.
+
+.. important::
+   The following is based on Odoo's default recruitment pipeline. Be advised that if
+   :ref:`modifications are made <recruitment/customize-stages>` to the pipeline, the following will
+   differ.
+
+.. _recruitment/new:
+
+New
+===
+
+All applicants appear in the :guilabel:`New` stage, whether submitted online or if the applicant is
+:doc:`manually entered by a recruiter <add-new-applicants>`.
+
+When the applicant's card is created, Odoo automatically populates the
+:guilabel:`Subject/Application`, the applicant's :guilabel:`Name`, :guilabel:`Email`,
+:guilabel:`Mobile` number, and :guilabel:`LinkedIn Profile` on the applicant's card. This
+information is required when applying for a job position, by default.
+
+.. note::
+   If the website application form is modified, different fields may be populated based on what
+   information is requested on the website.
+
+If the applicant entered any information in the :guilabel:`Short Introduction` section of the online
+application, it populates the :guilabel:`Application Summary` tab at the bottom of the applicant's
+card.
+
+Resumé
+------
+
+If a resumé was attached to the online application, it appears in the attachments section of the
+chatter, and is also stored in the *Documents* application. To find the recruitment documents,
+navigate to the main :menuselection:`Documents` dashboard, and click the :guilabel:`Recruitment`
+folder on the left-hand side. All recruitment documents are stored within that folder.
```

Comment:
I double checked right now, and I checked in the database I made for my recruitment tour. In a blank database, it does go to the Recruitment folder. I honestly do not know why it does not do that in Runbot, unless they made some modifications.

This is why my docs take forever, because  always have to compare to a blank/fresh DB and the runbots, since there are so many configs in Runbot =D! But thank you as always for double checking how everything works- I really appreciate it!